src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -34,7 +36,7 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         style={[
           styles.containerCard,
           {
-            backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
+            backgroundColor: isSelect ? anonColorCondition : COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in color based on the `isAnon` prop is a good addition for visual distinction between anonymous and non-anonymous users. However, it's important to ensure that these colors are accessible (i.e., they provide sufficient contrast) to all users, including those with visual impairments.



<!-- This is an auto-generated comment by OSS CodeRabbit -->